src/runtime/runtime.ts

```diff
@@ -56,7 +57,7 @@ let runtime: Runtime | null = null;
 // To start with, this class will simply hide the runtime classes that are
 // currently imported by ArcsLib.js. Once that refactoring is done, we can
 // think about what the api should actually look like.
-export class Runtime {
+export class Runtime implements RamDiskMemoryProvider {
```

Comment:
Alternatively we could have it hold one and provide it, but this seems sufficient and avoids overmuch providing of providers.